### PR TITLE
Added jQAssistant SonarQube Plugin 1.12.0

### PR DESCRIPTION
**Pull request by DirkMahler**

### CODE REVIEW

File: jqassistant.properties [hunk 1]
```diff
@@ -1,15 +1,21 @@
 category=External Analyzers
 description=Analyze issues reported by jQAssistant.
 homepageUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin
-archivedVersions=1.9.0,1.10.0
-publicVersions=1.11.0
+archivedVersions=1.7.3,1.8.1,1.8.2,1.9.0,1.10.0,1.11.0
```
Comment (ganncamp) [inline jqassistant.properties:4]: You're adding a lot of version numbers back into this list when the metadata for those versions no longer exists in the file. I'm pretty sure the job will fail with these missing versions. They were removed from the file because they only supported versions that are past EOL. Please revert the changes an only add the newly-archived version.

**Review by ganncamp — CHANGES_REQUESTED** [review]

File: jqassistant.properties (hunk 1 above)
Comment (DirkMahler) [reply jqassistant.properties]: The outdated versions are removed now.

**Review by ganncamp — APPROVED** [review]
